### DISCUSSION TITLE
Not able to pay with test cards

### DISCUSSION THREAD

**Discussion by danilo-olacerda** [mercadopago/sdk-js#176] [Q&A]
I'm trying for a long time use 2 tests accounts to simulate a purchase but the card is always declined

![image](https://github.com/mercadopago/sdk-js/assets/79648134/a9936ed1-8002-4a1d-8916-eebcb23e3048)

**Comment by romerosilva-meli** [comment]
Hi @danilo-olacerda thanks for reaching out to us. The name for an approved payment is only 'APRO' and not 'APRO APRO'. Can you test and see if this fixes the problem?

**Reply by danilo-olacerda** [reply]
Hello @romerosilva-meli thank you for the response.

The field does not let me use just "APRO" as value, also i made it as "APRO APRO" cause was in the docs.

![image](https://github.com/mercadopago/sdk-js/assets/79648134/ce42f087-f2d4-4e29-9e60-965723888c41)

![image](https://github.com/mercadopago/sdk-js/assets/79648134/7fa62e71-4383-458b-b627-1c4725d6250e)

**Comment by lucmkz** [comment]
Hello @danilo-olacerda
Thank you for your contact.

Question, did you build this interface yourself?
I wonder, because this SDK doesn't do any kind of validation in relation to the name entered.

So I would like a little more context from your application.

Also to help me simulate the scenario you are having, can you please share some paylaod information with me?

First, I need to see the payload you are sending in the "createCardToken" method
And then on your backend I need to see the payload that you're sending to the payments creation endpoint.

Ps. Don't forget to remove all your keys and access tokens.

Lucas.

**Comment by lucmkz** [comment]
due to lack of feedback I am ending the discussion, if necessary, please open a new one.

Lucas
